app/views/budgets/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= render 'shared/navbar', {text:"Budgets", path:budgets_path, visible:"visible"}%>
+
+<section class="cont container-fluid">
+  <div>
+    <% @budgets.each do |budget|%>
+      <div class="head d-flex justify-content-between align-items-center p-4">
+        <div class="budget d-flex gap-3">
+          <img src= <%= budget.icon %> height="75" width="100" alt="<%= budget.name %> icon" />
+          <div class="budget_description">
+            <h2><%= link_to budget.name, budget %></h2>
+            <p><%= budget.created_at.strftime("%B %d, %Y") %></p>
+          </div>
+        </div>
+        <div class="total_amount">
+          <p><%= "$#{budget.expenses.sum(:amount)}" %></p>
+        </div>
+      </div>
+    <% end %>
+  </div>
+  <div class="plus d-flex justify-content-end p-5">
+    <%= link_to new_budget_path do %>
+      <i class="bi bi-plus-circle-fill"></i>
+    <% end %>
+  </div>
```

Comment:
- [ ] When there are no budgets created yet, the home page is just empty and there are no navigation options unless I manually update the URL. Please fix this issue and update it on the Live Demo Link of the app. 🙏 
![GrowRich](https://github.com/Kwaku28/Grow-Rich/assets/47625407/17e8a1f2-1d60-4201-81dc-fbf2a9ea7a7b)

- [ ] Also, the design is way too different from the one you presented in the video. Please inspect the issue on the deployment and fix it. 🙏

Reply:
- [ ] Splash screen
![Splash](https://github.com/Kwaku28/Grow-Rich/assets/113942699/d92bdfca-62a2-41c4-b54e-f446c6a37601)

- [ ] Home screen
![home](https://github.com/Kwaku28/Grow-Rich/assets/113942699/5bc0e2b9-4ffe-461d-9657-106fd2a94235)